### ISSUE TITLE
Duplicated files in ZIP archive when unpacking

### ISSUE THREAD

**Issue by tempuserr** [maestrith/AHK-Studio#156]
Hi,

There are duplicated files in archive with source code from Master Branch and last release 1.005.00 source code zip:
- **README.md** (size 59 bytes) + **Readme.md** (size 23 bytes)
- lib\**Commands.xml** (size 41 777 bytes) + lib\**commands.xml** (size 43 661 bytes)

All 4 files are in the same zip archive, so extracting programs ask for overwrite each other.

**Comment by maestrith** [comment]
> README.md (size 59 bytes) + Readme.md (size 23 bytes)
> lib\Commands.xml (size 41 777 bytes) + lib\commands.xml (size 43 661 bytes)
Removed them. Thanks for that.  I get so lost in updates that I forget that there are duplicates.
>There is no version number in version linked from your web page (title of About dialog in AHK-Studio is without version number).
Which site so I can fix it.  maestrith.com or the AutoHotkey forum? 1.005.XX+ is going to be the most recent/stable versions

**Comment by tempuserr** [comment]
"Webpage" I mean maestrith.com - this link to .ahk file from Master Branch (https://raw.githubusercontent.com/maestrith/AHK-Studio/master/AHK-Studio.ahk). It is same as when I select on _Git > Code > Clone or download > Download ZIP_.

And in this source code near beginnig is this line of code:
`	Setup(11),Hotkeys(11,{"Esc":"11Close"}), ;auto_version
`

Contrary on [this](https://github.com/maestrith/AHK-Studio/releases) page with 1.005.00 version in .ahk file there is:
`	Setup(11),Hotkeys(11,{"Esc":"11Close"}), Version:="**1.003.31**"
`

**Comment by maestrith** [comment]
oh boy...that's messed up again. Thanks for pointing that out.

**Comment by maestrith** [comment]
*Should* be fixed

**Comment by tempuserr** [comment]
There is 1.005.06 now.
Thanks!

**Comment by maestrith** [comment]
yep. I need to fix the way that Studio handles the tags.  It is still stuck on 1.005.00

**Comment by derz00** [comment]
I have noticed that version numbers haven’t made much sense in Studio. But never bothered to bother you. :P

**Comment by maestrith** [comment]
I could have sworn that I removed them before... I deleted them again.  Thanks for the heads up.  Hopefully they stay deleted :(

**Comment by thakyZ** [comment]
@maestrith I am sorry to announce, README.md and Readme.md are still in existence...
EDIT: it would appear commands too...
